subnet/local/README_local.md

```diff
@@ -0,0 +1,16 @@
+# Subnet Local Deployment
```

Comment:
Rename the file, just call `README.md` will do. Your dir name has said same thing